### PR TITLE
Cleaned up the source code with pyflakes

### PR DESCRIPTION
**Pull request by wetneb**
- some PEP8 naming fixes
- fixed all pyflakes warnings except in dissemin/ and notifications/
- removed some unused code

### CODE REVIEW

File: import_researchers.py [hunk 1]
```diff
@@ -3,18 +3,15 @@
 
 from codecs import open
 
-from backend.create import *
-from papers.models import *
+from papers.models import Researcher, Department
```
Comment (RaitoBezarius) [inline import_researchers.py]: Imports can be sorted.

*PyImportSortBear, severity NORMAL, section `python`.*

The issue can be fixed by applying the following patch:

```diff
--- a/import_researchers.py
+++ b/import_researchers.py
@@ -3,7 +3,8 @@
 
 from codecs import open
 
-from papers.models import Researcher, Department
+from papers.models import Department
+from papers.models import Researcher
 from papers.name import normalize_name_words
 
 last_name_f = 0
```

Comment (RaitoBezarius) [inline import_researchers.py]: Imports can be sorted.

*PyImportSortBear, severity NORMAL, section `python`.*

The issue can be fixed by applying the following patch:

```diff
--- a/import_researchers.py
+++ b/import_researchers.py
@@ -3,7 +3,8 @@
 
 from codecs import open
 
-from papers.models import Researcher, Department
+from papers.models import Department
+from papers.models import Researcher
 from papers.name import normalize_name_words
 
 last_name_f = 0
```

Comment (RaitoBezarius) [inline import_researchers.py]: Imports can be sorted.

*PyImportSortBear, severity NORMAL, section `python`.*

The issue can be fixed by applying the following patch:

```diff
--- a/import_researchers.py
+++ b/import_researchers.py
@@ -3,7 +3,8 @@
 
 from codecs import open
 
-from papers.models import Researcher, Department
+from papers.models import Department
+from papers.models import Researcher
 from papers.name import normalize_name_words
 
 last_name_f = 0
```

Comment (RaitoBezarius) [inline import_researchers.py]: Imports can be sorted.

*PyImportSortBear, severity NORMAL, section `python`.*

The issue can be fixed by applying the following patch:

```diff
--- a/import_researchers.py
+++ b/import_researchers.py
@@ -3,7 +3,8 @@
 
 from codecs import open
 
-from papers.models import Researcher, Department
+from papers.models import Department
+from papers.models import Researcher
 from papers.name import normalize_name_words
 
 last_name_f = 0
```

Comment (RaitoBezarius) [inline import_researchers.py]: Imports can be sorted.

*PyImportSortBear, severity NORMAL, section `python`.*

The issue can be fixed by applying the following patch:

```diff
--- a/import_researchers.py
+++ b/import_researchers.py
@@ -3,7 +3,8 @@
 
 from codecs import open
 
-from papers.models import Researcher, Department
+from papers.models import Department
+from papers.models import Researcher
 from papers.name import normalize_name_words
 
 last_name_f = 0
```

Comment (RaitoBezarius) [inline import_researchers.py]: Imports can be sorted.

*PyImportSortBear, severity NORMAL, section `python`.*

The issue can be fixed by applying the following patch:

```diff
--- a/import_researchers.py
+++ b/import_researchers.py
@@ -3,7 +3,8 @@
 
 from codecs import open
 
-from papers.models import Researcher, Department
+from papers.models import Department
+from papers.models import Researcher
 from papers.name import normalize_name_words
 
 last_name_f = 0
```

Comment (RaitoBezarius) [inline import_researchers.py]: Imports can be sorted.

*PyImportSortBear, severity NORMAL, section `python`.*

The issue can be fixed by applying the following patch:

```diff
--- a/import_researchers.py
+++ b/import_researchers.py
@@ -3,7 +3,8 @@
 
 from codecs import open
 
-from papers.models import Researcher, Department
+from papers.models import Department
+from papers.models import Researcher
 from papers.name import normalize_name_words
 
 last_name_f = 0
```

**Comment by wetneb** [comment]
@RaitoBezarius: didn't manage to run coala on my VM (Python 3…), feel free to run it on this branch before merging. Also it would be nice if new pyflakes warnings introduced by the PRs could be spotted (not sure how to do that in a clean way. add a script on travis?)

**Comment by RaitoBezarius** [comment]
@wetneb No problem, I'm going to take a look to it!

**Pull request merged**